common/cache/lru.go

```diff
@@ -62,6 +63,10 @@ type (
 		value      interface{}
 		refCount   int
 	}
+
+	cacheSizeGetter interface {
+		CacheSize() int
+	}
```

Comment:
i would suggest 

```go
itemSizeGetter interface {
	ItemSize() int
}
```

Reply:
I think CacheSize() is more intuitive here to claim this method is for evaluate cache size.

Reply:
I think this interface should be exported, defined in `cache.go` and properly documented.